.gitignore

```diff
@@ -1,9 +1,10 @@
 .DS_Store
-**/*/__pycache__
+**/__pycache__

```

Comment:
Was I wrong about this?

Reply:
I'm not sure, I was having trouble with `tests/pycache` not being ignored, but maybe I made a mistake, I'll have to look at it again